changelog.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 2.11.0~rc2
+
+* Address feedback on source strings (#7370)
+* Improve noble migration check script output (#7369)
+* Run securedrop-remove-packages hourly instead of daily (#7377)
+* Allow apache2 to read /etc/securedrop-noble-migration.json in apparmor (#7378)
```

Comment:
Fine for an RC changelog; I suggest clarifying for the final release, but take what's useful and leave the rest:

```suggestion
* Address translators' feedback on UI strings (#7370)
* Improve output of Ubuntu Noble pre-migration check script (#7369)
* Run `securedrop-remove-packages` hourly instead of daily (#7377)
* Allow `apache2` to read `/etc/securedrop-noble-migration.json` under AppArmor (#7378)
* Remove stray Ubuntu file `/etc/apt/apt.conf.d/zzzz-temp-installer-unattended-upgrade` if it exists (#7380)
```

Reply:
Thanks, squashed your changes in.